### ISSUE TITLE
I letter too close to tick symbol

### ISSUE THREAD

**Issue by superbonaci** [BiglySoftware/BiglyBT#3019]
Version
```
Java 20.0.1 (64 bit)
  Homebrew
/opt/homebrew/Cellar/openjdk/20.0.1/libexec/openjdk.jdk/Contents/Home

SWT v4944r26, cocoa, zoom=200, dpi=72
Mac OS X v13.5, aarch64 (64 bit)
B3.4.0.0/4 az3
```
Issue screenshot

<img width="901" alt="BiglyBT-tick-overlaps" src="https://github.com/BiglySoftware/BiglyBT/assets/8339596/00a44bfc-f03c-4b1b-b523-87a0d1f19b5b">

**Comment by parg** [comment]
That's a weid one - some kind of glitch in the SWT UI library.

**Comment by superbonaci** [comment]
Can you reproduce the issue?

**Comment by parg** [comment]
No

**Comment by superbonaci** [comment]
I need to fix the other bug first.

**Comment by ferdnyc** [comment]
Yeah, I can't reproduce on Linux, either, even by making the window super-narrow... the text just wraps (with proper spacing).

![image](https://github.com/BiglySoftware/BiglyBT/assets/538020/1b3e19f1-9844-443e-8cbb-1ad420726271)

**Comment by o0nd7ots** [comment]
3.4.0.0 from Flathub on Linux also works fine
![image](https://github.com/BiglySoftware/BiglyBT/assets/124466962/27bc5d4a-5773-4996-8b87-df508ad49266)

**Comment by superbonaci** [comment]
Maybe you could use a virtual machine or something to try macos

**Comment by ferdnyc** [comment]
Setting up macOS in a VM is no small feat, it's _quite_ involved.

Fortunately, I've already spent those two weeks of my life.

Just-released BiglyBT 3.5.0.0, installed using the packaged installers from the official downloads, running under macOS 10.15.7 (latest version that will work in my VM without horrible graphical corruption):

![image](https://github.com/BiglySoftware/BiglyBT/assets/538020/9758baea-0737-4613-86a0-8ecda3c4cc11)

...As you can see, everything looks perfect. Resizing the window does not change that:

![image](https://github.com/BiglySoftware/BiglyBT/assets/538020/3d115987-312e-4c60-aea2-ca94b3a42c1f)

**Comment by ferdnyc** [comment]
That's with:
* the bundled Java 1.8.0_392
* bundled SWT v4942r22
* `zoom=100`, which is probably the most likely factor in why we're seeing different things.
